keps/sig-apps/sidecarcontainers.md

```diff
@@ -196,7 +197,12 @@ Older Kubelets that don't implement the sidecar logic could have a pod scheduled
 ## Design Details
 
 ### Test Plan
-//TODO
+* Units test in kubelet package `kuberuntime` primarily in the same style as `TestComputePodActions` to test a variety of scenarios.
+* New E2E Tests to validate that pods with sidecars behave as expected e.g:
```

Comment:
It would be good to have some tests that incorporate init containers and termination grace period as well.

Reply:
agreed, I've added some cases to cover that